### ISSUE TITLE
in Qnn in train_loader  i need to use larger batchsize 

### ISSUE THREAD

**Issue by VYRION-Ai** [DRA-chaos/Quantum-Classical-Hyrid-Neural-Network-for-binary-image-classification-using-PyTorch-Qiskit-pipeline#3]
train_loader = torch.utils.data.DataLoader(cifar_trainset, batch_size=1, shuffle=True)
how to do that
